src/components/DateRangePickerInput.jsx

```diff
@@ -132,11 +144,14 @@ export default class DateRangePickerInput extends React.Component {
       customInputIcon,
       customArrowIcon,
       phrases,
+      isFocused,
     } = this.props;
 
     const inputIcon = customInputIcon || (<CalendarIcon />);
     const arrowIcon = customArrowIcon || (<RightArrow />);
 
+    const screenReaderText = screenReaderMessage || phrases.keyboardNavigationInstructions;
```

Comment:
kinda curious about this - if someone were to add in a custom screen reader message here, then the user wouldn't hear about the keyboard navigation instructions, even if they're not mutually exclusive. Since the keyboard controls are not customizable, it seems like the keyboard instructions should always be present.